### ISSUE TITLE
Is there accompany music in the Dali or Dsing30 dataset?

### ISSUE THREAD

**Issue by YoungKang1222** [microsoft/muzic#38]
Hi! 
I try  the pdaugment in my accompanied mandarin song dataset, the ratio between training data and augmented data is 1.3:1, but the test accurracy reduces a little bit comparing to the setup without augmented data.

I guess is that the accompany influence the result?

thanks for replying!

**Comment by actuy** [comment]
Yes, the accompaniment may bring a larger gap. Have you combined the original training data and the augmented data? And the augmented data is from which speech dataset, maybe you can check the speech dataset and the results after MFA.
